tests/test_mobile_bitcoin_price.py

```diff
@@ -76,7 +76,7 @@ def test_mobile_bitcoin_price(test_mobile_obj):
 
     # Run  the test only if the options provided are valid.
     if options_obj.check_options(options):
-        test_mobile_obj = PageFactory.get_page_object("Zero mobile")
+        test_mobile_obj = PageFactory.get_page_object("Zero mobile", base_url=test_obj.base_url)
```

Comment:
This line is throwing an error: 

tests/test_mobile_bitcoin_price.py::test_mobile_bitcoin_price Exception when trying to run test:/root/project/tests/test_mobile_bitcoin_price.py
Python says:local variable 'test_obj' referenced before assignment

For now, can we just revert the changes within this mobile test alone?

Reply:
@qxf2 - I have reverted the change and pushed it.